schema/sql/nss.sql

```diff
@@ -23,7 +22,6 @@ create view nss_pgsql.passwd as
         uid,
         uid as gid
     from public.passwd;
-grant select on nss_pgsql.passwd to nss_pgsql;
```

Comment:
Leave these where they are and move the `create user "nss_pgsql";` to the top of the file